src/js/modules/accounts/components/forms/SignInForm.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import injectSheet from "react-jss";
+import { EMAIL_REGEX } from "../../../../constants";
+
+const styles = {
+  errorMessage: {
+    color: "red",
+  },
+  successMessage: {
+    color: "green",
+  },
+};
+
+const validate = formValues => {
+  const errors = {};
+  if (!formValues.email) {
+    errors.email = "Required";
+  } else if (!EMAIL_REGEX.test(formValues.email)) {
+    errors.email = "Invalid email address";
+  }
+  return errors;
+};
+
+const renderField = ({
```

Comment:
Isn't this identical to above? Why not make it into a component? Well, actually it already is one, but why not make it into a separate file?

Reply:
??

Reply:
@jkao1